core/src/main/proto/Models.proto

```diff
@@ -0,0 +1,21 @@
+// This file has been automatically generated for use by
```

Comment:
@juanpedromoreno 
Curious, is the goal to check in the proto files? or always generate the proto files as managed/generated source? or something else?

Reply:
Apart from generating it to be used in other microservices, the other main motivation is compatibility. Other clients (even in other languages) could use it. With `frees-rpc` we've covered the scala part, this would be a mechanism to interop with other microservices/platforms.

That said, ideally, the plugin could generate the proto file in the `compile` phase. Therefore, you wouldn't need to invoke the `protoGen` task manually. This is something I have not been able to make yet. As output, the proto file would be generated each time you recompile your code, putting it in the managed sources you set up.

Reply:
For example, by having the proto file available, we could use it to generate compatible services in a TypeScript frontend environment through a protoc plugin like this:

https://github.com/improbable-eng/ts-protoc-gen